### ISSUE TITLE
[semantic-issue] Add semantics for sklearn LogisticRegression

### ISSUE THREAD

**Issue by osryde** [spangea/Pyra#88]
**Description**
Notebook: 0-91-accuracy-score-for-4-models-lr-rc-gbc-rfc.py

**Minimal running example**
```
from sklearn.linear_model import LogisticRegression, RidgeClassifier

models = {'LG': LogisticRegression(), 'Ridgeclassifier': RidgeClassifier()}
```

**Links to documentation**
https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html

**Note (optional)**
Returns an instance of LogisticRegression
